docs/dapp/tools/remix.md

```diff
@@ -0,0 +1,126 @@
+
+![The initial example project in Remix - Ethereum IDE](../images/remix/remix01.png)
+
+## Contract Compilation
+
+1. Navigate to the **Solidity Compiler** tab
+2. Configure the compiler settings:
+
+   - Compiler version: **`0.8.24`**
+   - EVM version: **`paris`** (found under Advanced Configuration)
+
+3. Click `Compile 1_Storage.sol`
+
+:::info Compiler Version
+
+The Sapphire uses the [Rust Ethereum EVM][rust-evm]. This implementation is
+compatible with Solidity versions up to **0.8.24**. However, it does not yet
+support some transaction types introduced in Solidity **0.8.25**, such as those
+mentioned in [rust-ethereum/evm#277][revm-277], which are pending release.
+
+:::
+
+:::info EVM Version
+
+EVM versions after **paris** (shanghai and upwards) include the PUSH0 opcode which
+isn't supported on Sapphire.
+
+:::
+
+[rust-evm]:https://github.com/rust-ethereum/evm
+[revm-277]: https://github.com/rust-ethereum/evm/issues/277
+
+![Solidity compiler tab](../images/remix/remix02.png)
+
+## Contract Deployment
+
+1. Open the **Deploy and Run Transactions** tab.
+2. Select `Injected Web3` as environment.
+3. Accept in MetaMask the account connection to Remix.
+
+![MetaMask connection confirmation](../images/remix/remix03.png)
+
+4. Click `Deploy`.
+5. Review and confirm the transaction in MetaMask.
+
+![Metamask transaction confirmation](../images/remix/remix04.png)
+
+If everything goes well, your transaction will be deployed using the selected
+account in the MetaMask and the corresponding Sapphire Network.
+
+## Working with Confidential Features
+
+Note that Remix operates without a Sapphire client, meaning transactions and
+queries are unencrypted and unsigned by default. To make use of Sapphire's
+confidential features, refer to our [Quickstart Tutorial].
+
+[Quickstart Tutorial]: https://docs.oasis.io/dapp/sapphire/quickstart
+
+
+## Additional Tools
+
+### Sourcify Integration
+
+Verified contracts on Sourcify can be opened directly in Remix, enabling
+contract interaction without a dedicated frontend.
+
+:::info ABI-Playground
+
+Alternatively, consider using our [ABI-Playground] for contract interactions
+on Sapphire.
+
+:::
+
+![Sourcify verfied contract](../images/remix/sourcify.png)
+
+:::info Reproduce byte code
+
+When reproducing bytecode for verified contracts in Remix, you may encounter
+some known issues. See our detailed discussion in this [GitHub issue][issue].
+
+:::
+
+Should you have any questions, do not hesitate to share them with us on the
+[#dev-central Discord channel][discord].
+
+[localnet]: ./localnet.mdx
+[network page]: https://docs.oasis.io/dapp/sapphire/network#rpc-endpoints
+[Remix]: https://remix.ethereum.org
+[Remix documentation]: https://remix-ide.readthedocs.io/en/latest/
+[metamask]: ../../general/manage-tokens/README.mdx#metamask
+[discord]: https://oasis.io/discord
+[ABI-Playground]: https://abi-playground.oasis.io/
+[issue]: https://github.com/oasisprotocol/docs/issues/866#issuecomment-2387513649
```

Comment:
Hmm. This is one specific issue though?

Reply:
actually it's broken now totally? I can't open verified source code in Remix, even USDT on Ethereum just shows me then the default workspace in Remix. Does it work for you?

Reply:
removed the section for now, as there is an issue with sourcify to open contracts in remix